framework-core/src/main/java/org/daisy/pipeline/script/impl/parser/StaxXProcScriptParser.java

```diff
@@ -464,13 +464,13 @@ else if (elemsToIgnore == 0) {
 									else if (role.equals(Values.NAME)) {
 										reader.next();
 										dHolder.mShort = reader.peek()
-												.asCharacters().getData();
+												.asCharacters().getData().replaceAll("\\s\\s+","");
 									}
```

Comment:
@capitancambio  This will totally remove whitespace (not only normalize it), is it what's really intended? Can you plz clarify?
But if so, the regex can be simplified to just `\\s+`?

This applies to the 2 last changes too.